### PR TITLE
Updates to use Pipelines Beta

### PR DESCRIPTION
**Pull request by a-roberts**
Credits to @Megan-Wright for the PipelineResources fix bundled up in this commit

Note we still need to modify the examples/test resources, but this at least gives us a functional Dashboard to run with so we can easily make those changes going forward. Let's see how far this gets (I may need to update the integration tests).

There are plenty of v1alpha1 references in the Go code too - iDK why but things do work still, perhaps these will go when rc3 no longer exists and beta goes live.

# Changes
rc3 of Pipelines added as an override, API version overrides added too, pipeline0 modified to use v1beta1 type

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by a-roberts** [comment]
/retest

Errors seen with the dep ensure (i/o timeout, doubt it's caused by this)

**Comment by a-roberts** [comment]
Current problem
```

I0330 11:45:24.780] # github.com/tektoncd/dashboard/vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1
I0330 11:45:24.781] vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1/cluster_task_conversion.go:27:5: cannot use (*ClusterTask)(nil) (type *ClusterTask) as type apis.Convertible in assignment:
I0330 11:45:24.782] 	*ClusterTask does not implement apis.Convertible (missing ConvertFrom method)
I0330 11:45:24.782] vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1/pipeline_conversion.go:27:5: cannot use (*Pipeline)(nil) (type *Pipeline) as type apis.Convertible in assignment:
I0330 11:45:24.782] 	*Pipeline does not implement apis.Convertible (missing ConvertFrom method)
I0330 11:45:24.783] vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1/pipelinerun_conversion.go:27:5: cannot use (*PipelineRun)(nil) (type *PipelineRun) as type apis.Convertible in assignment:
I0330 11:45:24.783] 	*PipelineRun does not implement apis.Convertible (missing ConvertFrom method)
I0330 11:45:24.784] vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1/task_conversion.go:27:5: cannot use (*Task)(nil) (type *Task) as type apis.Convertible in assignment:
I0330 11:45:24.784] 	*Task does not implement apis.Convertible (missing ConvertFrom method)
I0330 11:45:24.784] vendor/github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1/taskrun_conversion.go:27:5: cannot use (*TaskRun)(nil) (type *TaskRun) as type apis.Convertible in assignment:
I0330 11:45:24.785] 	*TaskRun does not implement apis.Convertible (missing ConvertFrom method)
```

definitely fixed that so checking I picked up all of my changes (this one's around knative/pkg)

File: base/400-pipeline0-task.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```
Comment (AlanGreene) [inline base/400-pipeline0-task.yaml]: we'll need to update the spec too:

```yaml
spec:
  resources:
    inputs:
      - name: git-source
        type: git
  params:
    ...
```

Comment (a-roberts) [reply base/400-pipeline0-task.yaml]: Yep, thanks, I noticed that from your change https://github.com/tektoncd/dashboard/commit/080a81ef493f0e3780c5e1ffeea287fdff32f957 just now actually 🤞

**Comment by a-roberts** [comment]
image

😭

**Comment by a-roberts** [comment]
It's green 🐸

**Pull request merged**